spec/cb/clients/job_spec.rb

```diff
@@ -12,108 +12,6 @@
 
 module Cb
   describe Cb::Clients::Job do
-    describe '#search' do
```

Comment:
This backs my thinking: https://buildz.cbmtn.io/repository/download/SupportGems_RubyCbApi/21391:id/index.html#9cbe7cf53959fb76a93bfc657453d4f93f58889a